### PR TITLE
unify gitignore

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,47 @@
...
+msg_gen/
+srv_gen/
+msg/*Action.msg
+msg/*ActionFeedback.msg
+msg/*ActionGoal.msg
+msg/*ActionResult.msg
+msg/*Feedback.msg
+msg/*Goal.msg
+msg/*Result.msg
+msg/_*.py
```
Comment (mgruhler) [inline .gitignore:13]: all this msg and srv stuff is a leftover from rosbuild, isn't it? So shouldn't this be removed?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,47 @@
...
+
+# Generated by dynamic reconfigure
+*.cfgc
+/cfg/cpp/
+/cfg/*.py
```
Comment (mgruhler) [inline .gitignore:18]: same as above for dynamic reconfigure

File: .gitignore [hunk 3]
```diff
@@ -1 +1,47 @@
...
+*.wikidoc
+
+# eclipse stuff
+.project
+.cproject
+
+# qcreator stuff
+CMakeLists.txt.user
+
+srv/_*.py
```
Comment (mgruhler) [inline .gitignore:31]: and srv

File: .gitignore [hunk 4]
```diff
@@ -1 +1,47 @@
...
+*.pcd
+*.pyc
+qtcreator-*
+*.user
+
+/planning/cfg
+/planning/docs
+/planning/src
```
Comment (mgruhler) [inline .gitignore:39]: where is this from?

**Comment by fmessmer** [comment]
@ipa-mig 
Let's discuss `.gitignore` in https://github.com/ipa320/cob_calibration_data/pull/118

**Comment by fmessmer** [comment]
closing in favour of default ipa320 .gitignore

**Pull request closed without merging**